### PR TITLE
added feature to signMessage to check if the string is a filename

### PR DESCRIPTION
**Pull request by wanderer**

### CODE REVIEW

File: helpeth [hunk 1]
```diff
@@ -187,8 +187,17 @@ switch (command) {
     wallet = loadWallet()
 
     // NOTE: geth internally accepts any input and zero-pads it
-    var msg = ethUtil.sha3(ethUtil.toBuffer(argv.message))
-    console.log('Input message: ' + argv.message)
+    var msg = argv.message
+    try {
+      fs.statSync(msg)
```
Comment (axic) [inline helpeth]: Why do you need this before `readFileSync`?

**Comment by axic** [comment]
Maybe it would make sense adding a new method for keccak hashing (i.e. `helpeth sha3 <msg/file>` to be in line with web3.js) and changing `signMessage` to support a hash or a text message?

File: helpeth (hunk 1 above)
Comment (wanderer) [inline helpeth]: good point, it is redundant ill remove it

**Comment by wanderer** [comment]
possibly. But I would hold off on adding until someone actually needs that feature

**Pull request merged**